.github/workflows/release.yml

```diff
@@ -358,6 +373,10 @@ jobs:
         echo "Pushing ${{ steps.canonical_version.outputs.result }} to ${{ env.DOCKERHUB_REPO }} ..."
         docker tag localhost:5000/local-metabase:${{ steps.canonical_version.outputs.result }} ${{ env.DOCKERHUB_REPO }}:${{ steps.canonical_version.outputs.result }}
         docker push ${{ env.DOCKERHUB_REPO }}:${{ steps.canonical_version.outputs.result }}
+
+        echo "Pushing ${{ steps.short_canonical_version.outputs.result }} to ${{ env.DOCKERHUB_REPO }} ..."
+        docker tag localhost:5000/local-metabase:${{ steps.canonical_version.outputs.result }} ${{ env.DOCKERHUB_REPO }}:${{ steps.short_canonical_version.outputs.result }}
```

Comment:
The tag same image (canonical_version) `localhost:5000/local-metabase:${{ steps.canonical_version.outputs.result }}` as with a new tag (short_canonical_version) `${{ env.DOCKERHUB_REPO }}:${{ steps.short_canonical_version.outputs.result }}`

The rest of the commands should refer to the new short canonical output.